src/app/rundown-view/pipes/timer/timer.pipe.ts

```diff
@@ -0,0 +1,27 @@
+import { Pipe, PipeTransform } from '@angular/core'
+
+const POSITIVE_SIGN: string = '+'
+const NEGATIVE_SIGN: string = '-'
+
+@Pipe({
+  name: 'timer',
+})
+export class TimerPipe implements PipeTransform {
+  public transform(durationInMs: number): string {
+    const durationInSeconds: number = Math.floor(durationInMs / 1000)
+    const absoluteDurationInSeconds: number = Math.abs(durationInSeconds)
+    const seconds: number = absoluteDurationInSeconds % 60
+    const minutes: number = Math.floor((absoluteDurationInSeconds % 3600) / 60)
+    const hours: number = Math.floor(absoluteDurationInSeconds / 3600)
+    const sign: string = this.getSign(durationInSeconds)
+    return `${sign}${this.formatDigits(hours)}:${this.formatDigits(minutes)}:${this.formatDigits(seconds)}`
```

Comment:
We should really try to avoid doing manually time manipualtion since it's a field full of egdecases and many good libraries exits for it out there.

I will highly recommed us using the library `date-fns` for everything timing related.
I believe their method `formatDuration` will do what you are trying to achieve here (minus the positive/negative sign, but you can always prepend that).
https://date-fns.org/v2.30.0/docs/formatDuration

Reply:
Also, this Pipe is closely related to `TimestampPipe` in the form that is doing close to the same logic.
What are the differences between `TimerPipe` and `TimestampPipe`?
From their names alone I wouldn't know which one to use where.
This should be cleaned up.

Reply:
This will be changed in SOF-1680, the idea is for `Timer` to replace `Timestamp` and have the specific logic only for our own needs. I checked `date-fns` it does not seem to easily cover our usecases, but i'm not against using a library if we find one that fits.

Reply:
And what is the specific usecase?

Reply:
Just for the record: I'm very much not a fan of us having to do "time manipulation" manually.